src/main/java/framework/RestConfiguration.java

```diff
@@ -9,6 +9,6 @@
     public void configure(){
         RestAssured.baseURI= "http://jira.hillel.it";
         RestAssured.port =8080;
-        //RestAssured.basePath="/people";
+        //RestAssured.basePath="";
```

Comment:
Why do you need this?

Reply:
I tried to take out this class from the tests separately, as annotation
@BeforeTest